automated-comments/go/two-fer/strings_join_used_for_concatenation.md

```diff
@@ -0,0 +1,4 @@
+`strings.Join` is perfect if the data already comes in the form of a `[]string`.
+Take a look at `fmt.Sprintf` for this use case; it's a very powerful way of building up strings from multiple pieces.
+
+In case of simple concatenations, the `+` operator is a very fast alternative.
```

Comment:
I suggest removing this. When the student uses Sprintf, it will be given as a comment on line 5 of [automated-comments/go/two-fer/strings_builder_used_for_concatenation.md](https://github.com/exercism/website-copy/pull/1222/files#diff-da6983c41277b728eff789945e1250e4).

```suggestion
```